server/src/main/scala/io/delta/kernelsharedtable/DeltaSharedTableKernel.scala

```diff
@@ -741,7 +761,6 @@ private case class SharedTableSnapshot(
     version: Long,
     protocol: KernelProtocol,
     metadata: KernelMetadata,
-    // versionStats: Option[VersionStats],
```

Comment:
why are we removing versionStats?

Reply:
this was never being used and was always commented out. I am just doing a code cleanup since I noticed it now. I should have removed it in a previous PR, sorry about that

Reply:
could you double check with @charlenelyu-db on whether it's expected to be empty?

Reply:
VersionStats is used for shadow traffic and for the size and numFiles in metadata response (which are both not needed I believe). So I am just not bothering to set it when getting the snapshot. Confirmation on this would be appreciated though